projects/detr3d/mmdet3d_plugin/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .datasets.transforms import PhotoMetricDistortionMultiViewImage
+from .models.backbones.vovnet import VoVNet
+from .models.dense_heads.detr3d_head import Detr3DHead
+from .models.detectors.detr3d import Detr3D
+from .models.task_modules.hungarian_assigner_3d import HungarianAssigner3D
+from .models.task_modules.match_cost import BBox3DL1Cost
+from .models.task_modules.nms_free_coder import NMSFreeCoder
+from .models.layers.detr3d_transformer import (Detr3DCrossAtten,
+                                               Detr3DTransformer,
+                                               Detr3DTransformerDecoder)
```

Comment:
Add __all__ = []  like https://github.com/open-mmlab/mmdetection3d/pull/2173/files#diff-4760a2ce9277aca7d1f9a9d7e39b61433f7769a2dd8f3e3920196dc87aeaf216

Reply:
Actually the project works normally without 'all=[]'. I think it is redundant.
Why do we need this?

Reply:
Indeed, it works normally without `__all__=[]`.  `From .* import *'' initialize the contents in this package. `__all__` can control the exposed content.